explore/src/main/scala/explore/config/AdvancedConfigurationPanel.scala

```diff
@@ -0,0 +1,256 @@
+
+  val component =
+    ScalaFnComponent
+      .withHooks[Props]
+      .renderWithReuse { props =>
+        implicit val ctx = props.ctx
+
+        Form(size = Small)(
+          ExploreStyles.Compact,
+          ExploreStyles.AdvancedConfigurationGrid
+        )(
+          <.div(
+            ExploreStyles.ExploreForm,
+            ExploreStyles.AdvancedConfigurationCol1
+          )(
+            <.label("Grating", HelpIcon("configuration/grating.md")),
+            EnumViewOptionalSelect[ReuseView, Grating](
+              id = "override-grating",
+              value = props.scienceMode.zoom(overrideGratingLens)
+            ),
+            <.label("Filter", HelpIcon("configuration/filter.md"), ExploreStyles.SkipToNext),
+            EnumViewOptionalSelect[ReuseView, Filter](
+              id = "override-filter",
+              value = props.scienceMode.zoom(overrideFilterLens)
+            ),
+            <.label("FPU", HelpIcon("configuration/fpu.md"), ExploreStyles.SkipToNext),
+            EnumViewOptionalSelect[ReuseView, Fpu](
+              id = "override-fpu",
+              value = props.scienceMode.zoom(overrideFpuLens)
+            )
+          ),
+          <.div(ExploreStyles.ExploreForm, ExploreStyles.AdvancedConfigurationCol2)(
+            <.label("Binning", HelpIcon("configuration/binning.md")),
+            EnumViewOptionalSelect[ReuseView, (XBinning, YBinning)](
+              id = "explicitXBin",
+              value = props.scienceMode.zoom(unsafeDisjointOptionZip(explicitXBin, explicitYBin))
+            ),
+            <.label("Read Mode", HelpIcon("configuration/read-mode.md"), ExploreStyles.SkipToNext),
+            EnumViewOptionalSelect[ReuseView, (ReadMode, Gain)](
+              id = "explicitReadMode",
+              value =
+                props.scienceMode.zoom(unsafeDisjointOptionZip(explicitReadMode, explicitGain))
+            ),
+            <.label("ROI", HelpIcon("configuration/roi.md"), ExploreStyles.SkipToNext),
+            EnumViewOptionalSelect[ReuseView, Roi](
+              id = "explicitRoi",
+              value = props.scienceMode.zoom(explicitRoi)
+            )
+          ),
+          <.div(ExploreStyles.ExploreForm, ExploreStyles.AdvancedConfigurationCol3)(
+          ),
+          <.div(ExploreStyles.AdvancedConfigurationButtons)(
+            SequenceEditorPopup(
+              props.obsId,
+              trigger = Reuse.by(props.obsId)(
+                Button(
+                  size = Small,
+                  compact = true,
+                  clazz = ExploreStyles.VeryCompact,
+                  content = "View Sequence"
+                )
+              )
+            ),
+            Button(
+              size = Small,
+              compact = true,
+              clazz = ExploreStyles.VeryCompact,
+              content = "Simple Configuration",
+              icon = Icons.ChevronsLeft,
+              onClick = props.onShowBasic.value
+            )(^.tpe := "button")
```

Comment:
Could this be done with the `as` mechanism? It is just a detail

Reply:
Hmmm `as` is for changing the tag, but in this case we want to add an attribute (so that the button does not submit the form).